data_migration/models/res_partner.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    old_id = fields.Integer()
```

Comment:
@kanda999 I'm not sure what the plan is, but shouldn't this be in a separate module if we are eventually to uninstall this module once the migration is done?

Reply:
I thought this field should be retained after the transition is complete.

@AungKoKoLin1997 
Could you please separate this field in the new module?